docs/csharp/language-reference/statements/jump-statements.md

```diff
@@ -0,0 +1,114 @@
+---
+title: "Jump statements - C# reference"
+description: "Learn about C# jump statements: break, continue, return, and goto."
+ms.date: 12/06/2021
+f1_keywords:
+  - "break_CSharpKeyword"
+  - "continue_CSharpKeyword"
+  - "return_CSharpKeyword"
+  - "goto_CSharpKeyword"
+helpviewer_keywords:
+  - "break statement [C#]"
+  - "break keyword [C#]"
+  - "continue statement [C#]"
+  - "continue keyword [C#]"
+  - "return statement [C#]"
+  - "return keyword [C#]"
+  - "goto statement [C#]"
+  - "goto keyword [C#]"
+---
+# Jump statements (C# reference)
+
+The following statements unconditionally transfer control:
+
+- The [`break` statement](#the-break-statement): terminates the closest enclosing [iteration statement](iteration-statements.md) or [`switch` statement](selection-statements.md#the-switch-statement).
+- The [`continue` statement](#the-continue-statement): starts a new iteration of the closest enclosing [iteration statement](iteration-statements.md).
+- The [`return` statement](#the-return-statement): terminates execution of the function in which it appears and returns control to the caller.
+- The [`goto` statement](#the-goto-statement): transfers control to a statement that is marked by a label.
+
+## The `break` statement
```

Comment:
There are a few other cases here that could be explored. This may be another case where not qualifying a statement can be clarifying. I'm not sure yet.

- What about nested `switch` statements? A `break` only exits the inner-most switch. 
- What about a `break` inside a loop in the statements for a *switch-condition*. The `break` will exit the loop, but not the `switch` statement.
- Any level of nesting of those conditions, among others.

I don't have alternative, and it may not be necessary, but it's worth thinking through. From customer questions, the nested loops example *is* important.

Reply:
From the current feedback to the `break` article, I've concluded that nested loops and `switch` inside a loop are worth mentioning. As for other variants, I assume (1) they are less frequent (and maybe should be refactored), (2) may be reasoned about by analogy from the existing examples. So, I don't think the article should explore other cases. What do you think @BillWagner?